### DISCUSSION TITLE
What is the Schema Limit for the AI SQL Editor?

### DISCUSSION THREAD

**Discussion by actraiser** [supabase/supabase#16401] [Questions]
I enabled/opted-in to the new AI SQL Editor but even simple prompts will return

```
Your database metadata is too large for Supabase AI to ingest
```

What are the exact limitations here?  I have 37 tables, most of them with very few columns, so I don't consider my schema over-complicated or even large.

Greets
-act

**Comment by saltcod** [comment]
Don't have a precise answer for you tbh. We need to look at this — tracking here:
https://github.com/orgs/supabase/discussions/19594#discussioncomment-7863890

We basically need to handle any size schema, since no one has the same size db

I'll close this one in the meantime — thank you!
